lib/src/r_tree/non_leaf_node.dart

```diff
@@ -124,16 +124,26 @@ class NonLeafNode<E> extends Node<E> {
     if (nonLeafParent == null) return;
 
     var newLeafNode = LeafNode<E>(this.branchFactor);
-    newLeafNode.include(this);
```

Comment:
`_convertToLeafNode` was copying the MBR, which is unnecessary since conversion implies there are no children and thus null/0x0 bounds.